packages/devtools_app/lib/src/screens/vm_developer/object_inspector/object_store.dart

```diff
@@ -49,7 +49,10 @@ class _ObjectColumn extends ColumnData<ObjectStoreEntry>
     VoidCallback? onPressed,
   }) {
     return VmServiceObjectLink(
-      object: data.value,
+      // TODO(srawlins): What type is `data` at runtime? If cast to `int`, no
+      // tests fail, but that can't be right...
+      // ignore: avoid-dynamic
+      object: (data as dynamic).value,
```

Comment:
@bkonyi can you comment on this?

Reply:
I'll land for now; happy to revisit when @bkonyi comments.